From ae6dda610201c6bbd4a04c05764e2ccd0cfacae8 Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Fri, 2 May 2014 14:36:40 -0400
Subject: [misc] mei: fix format compilation warrning on 32 bit architecture

Message-id: <1399041470-21330-8-git-send-email-prarit@redhat.com>
Patchwork-id: 79293
O-Subject: [RHEL7.1 PATCH BZ 1088953 07/77] mei: fix format compilation warrning on 32 bit architecture
Bugzilla: 1088953
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1088953

commit e19555ce893f7567c7a72f91dafe6bdb93f0198f
Author: Tomas Winkler <tomas.winkler@intel.com>
Date:   Mon Sep 2 03:11:03 2013 +0300

    mei: fix format compilation warrning on 32 bit architecture

    hbm.c: In function mei_hbm_me_cl_allocate:
    hbm.c:52:212: warning: format %zd expects argument of type signed size_t but argument 4 has type long unsigned

    Signed-off-by: Tomas Winkler <tomas.winkler@intel.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/misc/mei/hbm.c b/drivers/misc/mei/hbm.c
index cdd0dff..0343a23 100644
--- a/drivers/misc/mei/hbm.c
+++ b/drivers/misc/mei/hbm.c
@@ -49,7 +49,7 @@ static void mei_hbm_me_cl_allocate(struct mei_device *dev)
  kfree(dev->me_clients);
  dev->me_clients = NULL;
 
- dev_dbg(&dev->pdev->dev, "memory allocation for ME clients size=%zd.\n",
+ dev_dbg(&dev->pdev->dev, "memory allocation for ME clients size=%ld.\n",
   dev->me_clients_num * sizeof(struct mei_me_client));
  /* allocate storage for ME clients representation */
  clients = kcalloc(dev->me_clients_num,
-- 
1.7.1